### DISCUSSION TITLE
Running PlutoSliderServer offline

### DISCUSSION THREAD

**Discussion by ametalci** [JuliaPluto/PlutoSliderServer.jl#140] [Q&A]
Hi, 
First of all thank you for this unique and amazing tool you have made allowing us to serve interactive Pluto notebooks, please keep up the good work!

One problem I could not find any solution is that :

1. any Pluto notebook served by PlutoNotebook.jl is not accessible if the client does not have any means of internet access. 
2. even the simplest hello world notebook (only added Pluto and PlutoUI packages) served tries to reach some links such as css, js files 
3.  since there is no internet access only a white screen is shown on browsers of the clients
4.  tested on Edge and Firefox browsers
5. the same problem does not occur if the internet access is restored on client side
6.  I am using default settings (ip, port, etc)
7. one of the links I could capture as an example is "- https://cdn.jsdelivr.net/gh/fonsp/Pluto.jl@0.19.43/frontend-dist/editor.b8733d72.js"

![image](https://github.com/JuliaPluto/PlutoSliderServer.jl/assets/601980/3f7e596a-6dd5-4fe6-a221-7eec59c82efd)

Has anyone encountered such an issue? 

Note: I would like to try copying css files next if I can find the correct location to put them into ...

Thanks.

**Comment by fonsp** [comment]
Sorry for the late reply! I missed your message before.

We are working on this in https://github.com/fonsp/Pluto.jl/pull/2802

**Reply by ametalci** [reply]
Thank you...
